python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -548,3 +575,28 @@ def successiveOverRelax(self, source, maxIter=None, eLevel=None):
             self.log.info("Success: SuccessiveOverRelaxation converged in %s iterations."
                           "\noutError: %s, inError: %s", nIter//2, outError, inError*eLevel)
         return func[1: -1, 1: -1]
+
+    @staticmethod
+    def extractCalibDate(calib):
```

Comment:
Move this function to the `utils.py` file, and call that in these two places.  It might be worth moving it to `isrFunctions.py` in the future, but that can be done on a future ticket (https://jira.lsstcorp.org/browse/DM-42622).

Reply:
This function also appear [here](https://github.com/lsst/ip_isr/blob/main/python/lsst/ip/isr/isrTask.py#L399) as a static method of `isrTask`. Should I also remove it from there and import it into `isrTask.py`?